assets/js/notification/index.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
+
+/**
+ * Display a notification explaining you're not logged yet,
+ * Clicking on it will open the authentication page.
+ */
+module.exports.displayNotLogged = function notLoggedNotification(site) {
+  var notificationId = "not_logged-" + site.name;
+  var options = {
+    type: "basic",
+    title: "Setup AnyFetch!",
+    message: "Click to sign-in and start using AnyFetch on " + site.name + "!",
+    iconUrl: "/images/icons/extension/icon128.png",
+    isClickable: true,
+  };
+  chrome.notifications.create(notificationId, options, function() {});
+
+  chrome.notifications.onClicked.addListener(function(_notificationId) {
+    if(notificationId === _notificationId) {
+      chrome.tabs.create({url: '/first-run.html'});
+    }
+  });
+
+  // Remove from notification center
+  setTimeout(function() {
+    chrome.notifications.clear(notificationId, function() {});
+  }, 10000);
```

Comment:
So isn't this cooldown a little low ? It means that if I navigate every 7 seconds I'll get a nasty notification every 15 seconds... I know we are trying to force the user, but it looks like spam for me here